src/pages/DulReview.js

```diff
@@ -37,31 +39,19 @@ class DulReview extends Component {
     const voteId = this.props.match.params.voteId;
     const consentId = this.props.match.params.consentId;
 
-    Votes.find(consentId, voteId).then(
-      vote => {
-        this.setState({
-          voteId: voteId,
-          vote: vote
-        });
-      }
-    );
+    const votesPromise = Votes.find(consentId, voteId);
+    const electionPromise = Election.findElectionByVoteId(voteId);
+    const consentPromise = Consent.findConsentById(consentId);
 
-    Election.findElectionByVoteId(voteId).then(
-      election => {
-        this.setState({
-          election: election
-        });
-      }
-    );
+    const [votes, elections, consent] = await Promise.all([votesPromise, electionPromise, consentPromise]);
```

Comment:
Much cleaner syntax, thanks for this, but I still find the approach a little painful after having been spoiled with [scala for comprehensions with futures](https://alvinalexander.com/scala/how-use-multiple-scala-futures-in-for-comprehension-loop/)